### PR TITLE
Chore/update sitemaps and rss logic

### PR DESCRIPTION
**Pull request by MildTomato**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES/NO

## What kind of change does this PR introduce?

- I’ve standardized the dates/times, before RSS feeds were being outputted with whatever the local time was. This meant a post on 14th Feb was being listed as published on 13th Feb due to our wide working timezones. This PR also updated this, and forces all times to be UTC first.
- **! important ! - Most items in RSS are now a different date because they were wrong before**
- Sitemap updated

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

requires https://github.com/supabase/supabase/pull/29871

### CODE REVIEW

File: apps/www/lib/rss.tsx [hunk 1]
```diff
@@ -22,7 +22,10 @@ const generateRssItem = (post: any): string => {
   const encodedTitle = xmlEncode(post.title)
   const encodedPath = xmlEncode(post.path)
   const encodedDescription = xmlEncode(post.description)
-  const formattedDate = dayjs(post.date).utc().format('ddd, DD MMM YYYY HH:mm:ss [GMT]')
+  const formattedDate = dayjs(post.date)
+    .utcOffset(0, true)
```
Comment (fsansalvadore) [inline apps/www/lib/rss.tsx:26]: should we offset it to California time?

Comment (fsansalvadore) [reply apps/www/lib/rss.tsx:26]: -7

Comment (MildTomato) [reply apps/www/lib/rss.tsx:26]: sure!

Comment (MildTomato) [reply apps/www/lib/rss.tsx:26]: i'm not sure what is right or wrong here, we probably need to standardize the times in the blog posts as well

**Review by fsansalvadore — APPROVED** [review]

**Pull request merged**